testing/checkin/runtime/__init.js

```diff
@@ -3,10 +3,12 @@ import "ext:checkin_runtime/__bootstrap.js";
 import * as async from "checkin:async";
 import * as testing from "checkin:testing";
 import * as console from "checkin:console";
+import * as error from "checkin:error";
 import * as timers from "checkin:timers";
 import * as worker from "checkin:worker";
 testing;
 async;
+error;
```

Comment:
I must have missed that. Why do we need these here?

Reply:
These need to be imported and then ignored to avoid the "unused import" error from the ExtModuleLoader. I suppose I can add a lint ignore for them being unused.

I think that the `node:` modules don't require this because we manually export them in `01_require.js`.